bruno/io-lollipop/reservePubKey.bru

```diff
@@ -53,6 +53,7 @@ script:pre-request {
   const thumbprint = await jose.JWK.asKey(publicKey).then(key => key.thumbprint("SHA-256")).then(thumbprint => thumbprint.toString("base64url"));
   
   bru.setVar("pubKey", publicKey);
+  bru.setVar("privKey", privateKey);
   bru.setVar("encodedPrivateKey", encodedPrivateKey);
```

Comment:
this `encodedPrivateKey` field is not necessary anymore after 7218d0bdd538761773ae4f0336feafa3f7b5812c